chains/43113/0x0b9d5D9136855f6FEc3c0993feE6E9CE8a297846/contract-information.json

```diff
@@ -0,0 +1,9 @@
+{
+  "address": "0xB767287A7143759f294CfB7b1Adbca1140F3de71",
```

Comment:
The address here does not correspond to the address in the file path

Reply:
Good catch!  Fixed now